lib/jekyll/site.rb

```diff
@@ -57,6 +59,13 @@ def process
       render
       cleanup
       write
+      print_stats
+    end
+
+    def print_stats
+      if @config['profile']
+        puts @liquid_renderer.stats_table
+      end
```

Comment:
Personally I think this way is more readable, even if it takes up 2 more lines.  I do the same exact thing he did in methods like that.  I only suffix the if when there are more lines  to follow but if it's a boolean method like this on I will multi-line the boolean statement.